docs/src/modules/components/HighlightedCodeWithTabs.tsx

```diff
@@ -0,0 +1,144 @@
+import * as React from 'react';
```

Comment:
Fixed
@siriwatknp I noticed, you used `<Tab slot={{ root: StyledTab }} />` instead of doing `StyledTab = styled(Tab)`

Moving from the first to the second strategy fixed the bug. But I'm wondering if their is a reason to choose the first one?